kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/installation/vault/Vault.java

```diff
@@ -44,16 +44,19 @@ public class Vault {
     private static final String VAULT_CMD = "vault";
     private final String deploymentNamespace;
     private final String vaultRootToken;
+    private final boolean isOpenshiftCluster;
```

Comment:
naming - we normally use `is` for method name prefixes, not the name of the field.  suggest `openshiftCluster`